hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -316,14 +319,16 @@ instance HasTransition 'StIdle where
     ]
 
 instance ObserveTx 'StIdle 'StInitialized where
-  observeTx tx OnChainHeadState{networkId, ownParty, ownVerificationKey} = do
-    (event, observation) <- observeInitTx networkId ownParty tx
+  observeTx tx OnChainHeadState{networkId, peerVerificationKeys, ownParty, ownVerificationKey} = do
+    let allVerificationKeys = ownVerificationKey : peerVerificationKeys
+    (event, observation) <- observeInitTx networkId allVerificationKeys ownParty tx
```

Comment:
Yes, but this is very much context dependent... I started with all keys in the State, but, since the state also stored the ownVerificationKey, there was redundancy in the state which is room for hazards.

Within the observation function however, we care not about this distinction and all keys can be treated the same.. Not very consistent but,... I don't know, still preferable?